indico/core/db/sqlalchemy/protection.py

```diff
@@ -354,6 +354,33 @@ def update_principal(self, principal, read_access=None, full_access=None, roles=
                                        old_data=old_data, quiet=quiet)
         return entry
 
+    def get_manager_list(self, recursive=False):
+        from MaKaC.conference import Category
+        managers = {x.principal for x in self.acl_entries if x.has_management_role()}
+        if recursive and self.protection_parent:
+            # XXX: Remove this condition check when moving Category to new models
+            if isinstance(self.protection_parent, Category):
+                managers.update(x.as_new for x in self.protection_parent.getRecursiveManagerList())
+            else:
+                managers.update(self.protection_parent.get_manager_list(recursive=True))
+        return managers
+
+    def get_access_list(self):
```

Comment:
Hm, `self.get_manager_list()` would be a problem too... but I don't think you need to call this at all:

- for the case of `if self.is_self_protected` you return everyone in the ACL which includes managers (once you remove the `read_access` check)
- for the case of inheriting you get the access list from the parent which also includes managers